standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -888,7 +888,7 @@ public void cleanEmptyAbortedAndCommittedTxns() throws MetaException {
           */
         long lowWaterMark = getOpenTxnTimeoutLowBoundaryTxnId(dbConn);
 
-        String s = "SELECT \"TXN_ID\" FROM \"TXNS\" WHERE " +
+        String s = "DELETE \"TXN_ID\" FROM \"TXNS\" WHERE " +
```

Comment:
Do the delete Ids get returned in the result set of executeQuery?  Can we delete the rest of the code that collects these Ids and then appears to delete them ( a second time)?

Reply:
yes, done. thank you for review.